packages/patternfly-3/react-console/src/DesktopViewer/MoreInformation.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+class MoreInformation extends React.Component {
```

Comment:
same thing here w/ More Information @lizsurette  -  i don't know if we want a standard design for this in PF or not.

<img width="493" alt="screen shot 2018-09-21 at 3 50 33 pm" src="https://user-images.githubusercontent.com/4237045/45902951-1c391280-bdb6-11e8-800d-951121a4cd33.png">

Reply:
@priley86 @mareklibra this looks very close to an implementation for something like this pattern:
https://www.patternfly.org/pattern-library/forms-and-controls/expand-collapse-section/ 

I think it would be a useful component as long as the contents could be configured. I'd envision these ranging from paragraph text like in this example and maybe form elements for say an "Advanced Settings" section of a form.

Reply:
@lizsurette , actually it was already extracted to #613.
Once it gets merged, I'll reuse here.

Reply:
#613 just needs tests updated and it should be g2g 👍